xbmc/ServiceManager.h

```diff
@@ -72,6 +72,7 @@ class CServiceManager
   CPlatform& GetPlatform();
 
   PLAYLIST::CPlayListPlayer& GetPlaylistPlayer();
+  int init_level = 0;
```

Comment:
as i tried to explain in the commit message and the original PR description, this was meant to highlight the issue (using reference returns for things that is dynamically allocated). i just did what was necessary to make it work again and variable names was not a focus point ;)